modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/humanoid.dm

```diff
@@ -29,3 +29,9 @@
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	payday_modifier = 0.75
 	examine_limb_id = SPECIES_HUMAN
+
+/datum/species/humanoid/get_species_description()
+	return placeholder_description
+
+/datum/species/humanoid/get_species_lore()
+	return list(placeholder_lore)
```

Comment:
These should probably have something that says they don't have any lore, and for people to create their own or something like that, same for the "broad" species.